src/hugo/content/en-us/patterns/MVC/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Model-View-Controller Device Software Design"
+weight: 10
+aliases:
+  - /designs/MVC/
+summary: "recommendation for the design and structure of device software"
+---
+
+## Challenge
+
+IoT devices have a common set of concerns: 
+* report the current state of the device, attached sensors and actuators
+* processing control messages (both remote and local)
+* presenting status information 
+
+The design should also be easy to test and reuse across multiple projects, hardware platforms, languages, and SDKs. Some devices are 'one-way' only reporting state or taking action, while others are 'two-way,' performing both of these actions. The design should be generally composable to support all these uses as well as accommodate a flexible variety of peripherals, command sources, and interfaces. 
```

Comment:
s/design/pattern/ throughout, that way it's consistent with other pattern descriptions.

Reply:
hhmm... that may be a shortcoming then.  This article describes a combination of several "Design Patterns" into a Software Design. That's a pretty common industry construction.  Can change it.. but then the word 'pattern' becomes highly redundant, overloaded, and possibly overused.

Reply:
To cover both hardware and software patterns, what about the use of "design pattern"? We will keep the menu heading as "Pattern" to be concise, but then can change the other patterns to replace "pattern" with "design pattern".